src/PopOver.tsx

```diff
@@ -133,6 +134,19 @@ function PopOver(props: PopOverProps) {
     >
       <DialogTitle id="alert-dialog-title">
         <Stack direction="row" alignItems="center">
+          <Box flexGrow={1}>{props.title}</Box>
```

Comment:
The title is already defined below this code. This makes the popover have two titles:
![image](https://github.com/kingsawpdx/virtualFieldEnvironments/assets/25037249/6aecd6c4-03ec-47a9-9b86-3bbad1819066)